common/constants.js

```diff
@@ -212,6 +226,19 @@ module.exports = Object.freeze({
   ANNOTATION_KEYS: {
     STATE: 'state',
     OPTIONS: 'options',
+    RESULT: 'result'
+  },
+  APISERVER: {
+    ANNOTATION_NAMES:{
+      BACKUP: 'backup'
+    },
+    ANNOTATION_TYPES:{
+      BACKUP: 'default'
```

Comment:
Can use `RESOURCE_TYPES`, And i think resource type is defaultbackup in apiserver definition.